warehouse/query-core/src/test/java/datawave/query/CompositeQueryTest.java

```diff
@@ -13,11 +13,14 @@
 import org.junit.BeforeClass;
 import org.junit.ClassRule;
 import org.junit.Test;
+import datawave.query.parser.JavaRegexAnalyzer;
```

Comment:
As it stands right now, this whole file can be reverted--looks like the only current changes are added imports.